src/opendata/Procfile

```diff
@@ -0,0 +1 @@
+web: data7 run --no-reload
```

Comment:
Should we remove the `--no-reload` option as it will be the default in future Data7 releases?